README.md

```diff
@@ -140,17 +140,10 @@ Search UI can connect to **any** web based Search API. Read [the advanced README
 
 The `SearchProvider` object will tie all of your Components together so that they work as a cohesive application.
 
-It's where you configure your UI, acting as the state manager between "State" and "Actions".
-
-The overall flow is like this:
-
-```
-Components -> SearchProvider -> Connector -> Search API (App Search)
```

Comment:
I like this line tbh, I'm going to remake it as a part of my talk because I think it highlights the 3 pieces `search-ui` provides and allows you to emphasize that any of them are able to be swapped out for your own code whenever you like.

Reply:
Ah yes, let me see if I can work that back in at the top of this section.